derp/derp_client.go

```diff
@@ -6,35 +6,35 @@ package derp
 
 import (
 	"bufio"
-	"crypto/rand"
+	crand "crypto/rand"
```

Comment:
FYI: this is a convention I like to make it super clear to readers below when something's crypto/rand and not potentially math/rand.